app/models/student_work_group.rb

```diff
@@ -27,10 +27,19 @@ class StudentWorkGroup < ApplicationRecord
   belongs_to :work_group
 
   validate :user_role
+  validates :work_group, presence: true
```

Comment:
this shouldn't be necessary
since there is the relation `belongs_to :work_group` the vlidation is included in it

Reply:
got it